ios/build.sh

```diff
@@ -124,51 +121,28 @@ update_bundle ( ) {
 
         cd $base_dir
     fi
-
-    #Copy the updated bundle to our output folder.
-    cp -Rf $KMEI_RESOURCES/Keyman.bundle ${KMEI_OUTPUT_FOLDER}
 }
 
 # First things first - update our dependencies.
-
-if ! [ -d "${KMEI_OUTPUT_FOLDER}" ]; then
-    mkdir -p "${KMEI_OUTPUT_FOLDER}"
-fi
-
 update_bundle
 
 echo
 echo "Building KMEI..."
 
 #OTHER_CFLAGS=-fembed-bitcode is relied upon for building the samples by command-line.  They build fine within XCode itself without it, though.
 
-rm -r $KMEI_BUILD_PATH/${CONFIG}-iphoneos 2>/dev/null
-xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -target KME-iphoneos OTHER_CFLAGS=-fembed-bitcode \
-  -configuration $CONFIG
-assertFileExists $KMEI_BUILD_PATH/${CONFIG}-iphoneos/libKME-iphoneos.a
-
-rm -r $KMEI_BUILD_PATH/${CONFIG}-iphonesimulator 2>/dev/null
-xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -sdk iphonesimulator PLATFORM_NAME=iphonesimulator \
-  -target KME-iphonesimulator OTHER_CFLAGS=-fembed-bitcode -configuration $CONFIG
-assertFileExists $KMEI_BUILD_PATH/${CONFIG}-iphonesimulator/libKME-iphonesimulator.a
+rm -r $KMEI_BUILD_PATH/$CONFIG-universal 2>/dev/null
+BUILD_FRAMEWORK_PATH=$KMEI_BUILD_PATH/$CONFIG-universal/KeymanEngine.framework
```

Comment:
While not strictly necessary, mind putting the `BUILD_FRAMEWORK_PATH` variable definition with the three previously-mentioned lines?  It'd make for nicer organization.

Reply:
- [ ] I don't see a file for the KME-universal scheme/target, while we do have the files KME-iphoneos.scheme and KME-iphonesimulator.scheme.  Does the scheme lack a separate file, or did this one manage to avoid getting added to source control so far?

Reply:
Removing the schemes you mentioned. `.scheme` files specify run schemes and can be autocreated from the project targets. I didn't do any special customization so we shouldn't have to check in the new scheme.

Reply:
OK, I can see the new scheme config within Xcode itself on my machine; I guess the script is auto-managed within the project files instead of as a separate scheme.